cicecore/cicedyn/analysis/ice_history.F90

```diff
@@ -1427,396 +1531,451 @@ subroutine init_hist (dt)
-             "y component of atmospheric stress on sea ice", &
-             "none", c1, c0,         &
-             ns1, f_sistrydtop, avg_ice_present=.true., mask_ice_free_points=.true.)
-
-         call define_hist_field(n_sistrxubot,"sistrxubot","N m-2",ustr2D, ucstr,  &
-             "x component of ocean stress on sea ice", &
-             "none", c1, c0,         &
-             ns1, f_sistrxubot, avg_ice_present=.true., mask_ice_free_points=.true.)
-
-         call define_hist_field(n_sistryubot,"sistryubot","N m-2",ustr2D, ucstr,  &
-             "y component of ocean stress on sea ice", &
-             "none", c1, c0,         &
-             ns1, f_sistryubot, avg_ice_present=.true., mask_ice_free_points=.true.)
-
-         call define_hist_field(n_sicompstren,"sicompstren","N m-1",tstr2D, tcstr,  &
-             "compressive sea ice strength",                      &
-             "none", c1, c0,         &
-             ns1, f_sicompstren, avg_ice_present=.true., mask_ice_free_points=.true.)
+             "sea-ice thickness",                                      &
+             "volume divided by sea-ice area", c1, c0,                 &
+             ns1, f_sithick, avg_ice_present='final', mask_ice_free_points=.true.)
+
+         call define_hist_field(n_siage,"siage","s",tstr2D, tcstr, &
+             "age of sea-ice",                                     &
+             "none", c1, c0,                                       &
+             ns1, f_siage, avg_ice_present='final', mask_ice_free_points=.true.)
+
+         call define_hist_field(n_sisnthick,"sisnthick","m",tstr2D, tcstr, &
+             "snow thickness",                                             &
+             "snow volume divided by sea-ice area", c1, c0,                        &
+             ns1, f_sisnthick, avg_ice_present='final', mask_ice_free_points=.true.)
+
+         call define_hist_field(n_sitemptop,"sitemptop","K",tstr2D, tcstr, &
+             "surface temperature of sea-ice",                             &
+             "none", c1, Tffresh,                                          &
+             ns1, f_sitemptop, avg_ice_present='final', mask_ice_free_points=.true.)
+
+         call define_hist_field(n_sitempsnic,"sitempsnic","K",tstr2D, tcstr, &
+             "temperature at snow-ice interface",                            &
+             "surface temperature when no snow present", c1, Tffresh,        &
+             ns1, f_sitempsnic, avg_ice_present='init', mask_ice_free_points=.true.)
+
+         call define_hist_field(n_sitempbot,"sitempbot","K",tstr2D, tcstr, &
+             "temperature at ice-ocean interface",                         &
+             "none", c1, Tffresh,                                          &
+             ns1, f_sitempbot, avg_ice_present='init', mask_ice_free_points=.true.)
+
+         call define_hist_field(n_siu,"siu","m/s",ustr2D, ucstr, &
+             "x-component of sea-ice velocity",                  &
```

Comment:
hyphen should not be in x component, but it's okay (optional) in sea-ice here (similarly elsewhere -- see comment about nouns vs adjectives)

Reply:
In the CMIP table they use a hyphen.